input/pagecontent/authentication.md

```diff
@@ -0,0 +1,169 @@
+- The IT service provider’s identity, if involved.
+
+In small systems, these verifications can be managed through direct agreements and trusting relationships between entities (e.g., hospitals trusting each others staff and HR systems).
+However, in national-scale healthcare systems with many organizations and professionals, direct agreements become impractical.
+
+Also, traditional authentication topologies relying on a central trusted authority introduce single points of failure and may not scale well. Downtime of such a central authority can disrupt access for all dependent entities. Also, often these schemes have a use-case tailored design, providing a limited set of identity claims, making them inflexible for future use-cases. Adding new identity claims require these central authorities to expand their systems and governance, which is often a slow process.
+
+#### Requirements
+
+This IG is focused on establishing a robust authentication mechanism that scales in a national healthcare context. This national context requires a solution that is scalable, cost-effective, and user-friendly for healthcare professionals.
+Also, it must support various (not yet known) use-cases, including those without direct end-user involvement (e.g., automated systems).
+To create a robust and safe solution in a hostile global internet, the topology must avoid a single point of failure and support peer-to-peer trust, enabling direct interactions between entities without relying on the availability of third parties.
+
+To summarize, the authentication solution must:
+
+- Support portable identity claims issued by authoritative sources.
+- Support presenting a combination of claims from different authoritative sources in a single transaction.
+- Support use cases with and without direct end-user involvement.
+- Enable service providers to act on behalf of care organizations, with scope-limited authorization.
+- Allow care organizations to act on behalf of healthcare professionals, with time-limited authorization.
+- Be user-friendly for healthcare professionals.
+- Be flexible and adaptable to various healthcare use cases.
+- Avoid single points of failure.
+- Adhere to privacy-by-design and security-by-design principles.
+- Leverage existing identity solutions such as x.509 certificates where possible.
+- Be using existing standards and and technologies where possible.
+
+### Terminology
+
+Throughout this document the following terminology is used:
+
+- Entity: An actor in the system (person or organization)
+- Agent: A digital representation of an entity that acts on its behalf, also often called an identity wallet
+- Vendor: A party that offers Agents to its customers (care organizations and healthcare professionals)
+- Identity: A set of claims about an entity (person or organization) which is relevant in a specific context
+- Claim: A statement about an entity (e.g. name, role, affiliation)
+- Verifiable Claim: A claim that can be cryptographically verified
```

Comment:
for what purpose, authenticity (authentic source)? Or integrity (maybe not issued by the source authentic source, but by another trusted party, or even the entity itself)